mmv1/third_party/terraform/services/memcache/resource_memcache_instance_test.go

```diff
@@ -97,3 +98,61 @@ data "google_compute_network" "memcache_network" {
 }
 `, name, network)
 }
+
+func TestAccMemcacheInstance_deletionprotection(t *testing.T) {
+	t.Parallel()
+
+	prefix := fmt.Sprintf("%d", acctest.RandInt(t))
+	name := fmt.Sprintf("tf-test-%s", prefix)
+	network := acctest.BootstrapSharedServiceNetworkingConnection(t, "memcache-instance-update-1")
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckMemcacheInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMemcacheInstance_deletionprotection(prefix, name, network, "us-central1"),
```

Comment:
This function needs to set deletion protection to `true`. It's currently not, which is why the test is failing to see the expected error in step 3